include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -48,6 +48,12 @@
 #include "static_array.hpp"
 #include "extents.hpp"
 #include "trait_backports.hpp"
+#include "depend_on_instantiation.hpp"
```

Comment:
Where is it used?

Reply:
This was used in a static_assert that's been removed, it can probably be removed as well.